assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceTilesWidget/index.test.js

```diff
@@ -267,7 +267,36 @@ describe( 'AudienceTilesWidget', () => {
 		registry.dispatch( CORE_USER ).receiveGetAudienceSettings( {
 			configuredAudiences: [
 				'properties/12345/audiences/1',
-				'properties/12345/audiences/9',
+				'properties/12345/audiences/3',
```

Comment:
Hmm, this "should render when some configured audiences are matching available audiences" test is now identical to the "should render when all configured audiences are matching available audiences" test. It looks like we can simply remove this "should render when some..." test case.

Reply:
Removed ✅